imap_upstream.adoc

```diff
@@ -16,23 +16,29 @@ IMAP related configuration options are available:
 
 The available balancing algorithms are as follows:
 
-* ClientIpHash
+* AccountIdHash
 
-  Selects an IMAP server server based on a hash of the client IP address.
-  The hashing algorithm used is a simple modulus of the client's ip address as an integer with the number of
-   configured IMAP servers. This will always return the same 'index' for the same client ip address.
-
-   Note: Extra configuration is necessary if a NAT proxy is being utilized in the environment.
+  Selects an IMAP server server based on a hash of the account id of
```

Comment:
Not your change but "IMAP server server" sounds weird.  Guessing there is one to many "server"